tools/test-auth-negative.sh

```
@@ -281,39 +281,39 @@ test_oidc_callback_validation() {
     # Test 5: Invalid/corrupted state cookie
     log "Testing OIDC callback with invalid state cookie..."
-    response=$(curl $CURL_OPTS --cookie "OIDC_STATE=corrupted_state_value" \
+    response=$(curl $CURL_OPTS --cookie "SK_AUTH_HANDSHAKE=corrupted_state_value" \
         "${OIDC_CALLBACK_URL}?code=fake&state=fake")
-    assert_response_contains "Invalid state cookie rejected" "$response" "Session%20expired"
+    assert_response_contains "Invalid state cookie rejected" "$response" "authError=true"
 
     # Test 6: State mismatch (cookie vs URL param)
     log "Testing OIDC callback with state mismatch..."
-    response=$(curl $CURL_OPTS --cookie "OIDC_STATE=state_in_cookie" \
+    response=$(curl $CURL_OPTS --cookie "SK_AUTH_HANDSHAKE=state_in_cookie" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)